code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -166,6 +166,9 @@
 	..()
 	icon_state = "secglockfancy[chambered ? "" : "-e"][silenced ? "-s" : ""][stored_magazine ? "" : "-m"]"
 
+/obj/item/weapon/gun/projectile/sec/lockbox
+	mag_type = "/obj/item/ammo_storage/magazine/m380auto/empty"
```

Comment:
did you check if this can still be loaded with the alternate magazines?

Reply:
No they don't. Someone else or myself has to fix these issues. I have an idea in mind but that's for another PR. I can remove the guns for now I guess till said fix or leave them in.

Reply:
Just tested it. They cannot use the other ones. I can work on a fix for it today.

Reply:
might as well fix it now

Reply:
Issue resolved.